src/Avalonia.Base/Media/Fonts/FontFamilyKey.cs

```diff
@@ -41,10 +41,7 @@ public override int GetHashCode()
             {
                 var hash = (int)2166136261;
 
-                if (Source != null)
-                {
-                    hash = (hash * 16777619) ^ Source.GetHashCode();
-                }
+                hash = (hash * 16777619) ^ Source.GetHashCode();
```

Comment:
Makes me think that FontFamilyKey (and possibly FontFamily) can be changed into records and removing bunch of this code.
But outside of the scope of this PR.

Reply:
That is a good use of records when you aren't implementing hashing and equality already. I still prefer the `record class` syntax in that case as `record` seems like a total misnomer when used like this. That might be just me though.